package.json

```diff
@@ -0,0 +1,53 @@
+{
+  "name": "@reside-ic/vue-nested-multiselect",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/reside-ic/vue-nested-multiselect.git"
+  },
+  "license": "MIT",
+  "author": "reside-ic",
+  "homepage": "https://github.com/reside-ic/vue-nested-multiselect#readme",
+  "version": "0.0.15",
```

Comment:
This might be misleading if this is day 0 for this project...

Reply:
once all the reviews were done i was just gonna bump this to 1.0.0, does that sound alright?

Reply:
yep, sounds good